iam_builder/templates.py

```diff
@@ -98,6 +98,100 @@
             ]
         }
     ],
+    "cadet_deployer": [
+        {
+            "Sid": "GlueCatalogActions",
+            "Effect": "Allow",
+            "Action": [
+                "glue:Get*",
+                "glue:DeleteTable",
+                "glue:DeleteTableVersion",
+                "glue:DeleteSchema",
+                "glue:DeletePartition",
+                "glue:DeleteDatabase",
+                "glue:UpdateTable",
+                "glue:UpdateSchema",
+                "glue:UpdatePartition",
+                "glue:UpdateDatabase",
+                "glue:CreateTable",
+                "glue:CreateSchema",
+                "glue:CreatePartition",
+                "glue:CreatePartitionIndex",
+                "glue:BatchCreatePartition",
+                "glue:CreateDatabase"
+            ],
+            "Resource": [
+                "arn:aws:glue:*:*:schema/*",
+                "arn:aws:glue:*:*:database/*",
+                "arn:aws:glue:*:*:table/*/*",
+                "arn:aws:glue:*:*:catalog"
+            ]
+        },
+        {
+            "Sid": "AthenaActions",
+            "Effect": "Allow",
+            "Action": [
+                "athena:List*",
+                "athena:Get*",
+                "athena:StartQueryExecution",
+                "athena:StopQueryExecution"
+            ],
+            "Resource": [
+                "arn:aws:athena:*:*:datacatalog/*",
```

Comment:
For my understanding - should this be 'awsdatacatalog' instead of datacatalog? Or is awsdatacatalog the name of the glue catalogue itself, and multiple glue catalogues are stored under athena 'datacatalog' resource?

Reply:
So this refers to the internal IAM structure, `datacatalog`, which is the generic name for the resource type across all data catalogs in Athena. If we wanted to make it more specific, you could limit to `arn:aws:athena:*:*:datacatalog/awsdatacatalog`, but we've left it generic to be in line with current permissions, as the role works across accounts and so may need to target different catalogs from the generic at times.